### PR TITLE
feat: Show discussions in app

### PR DESCRIPTION
**Pull request by ChaituVR**
closes https://github.com/snapshot-labs/workflow/issues/192

## Summary:
- Topics in discussions page should redirect to new topic page
- Show a button in bottom to redirect to discourse 
image
- Same design as proposal page discussion
- Add topic title on top of topic page 
Untitled 2

### How to test:
- Go to http://localhost:8080/#/s:safe.eth/discussions/
- Click on different topics 

### TODO:
- [ ] Render HTML like in #688

### CODE REVIEW

File: apps/ui/src/views/Space/DiscussionsTopic.vue [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  }
+});
+</script>
+
+<template>
+  <div>
+    <div v-if="loading" class="p-4">
+      <UiLoading />
+    </div>
+    <div v-if="loaded" class="px-4 pl-6 pt-5">
```
Comment (wa0x6e) [inline apps/ui/src/views/Space/DiscussionsTopic.vue:58]: Why do we have more left padding than right ?

**Review by wa0x6e — APPROVED** [review]
tAck

**Comment by wa0x6e** [comment]
Added a few improvement:

- Make ui same as proposal discussion (centered content)
- Refresh page content when url change
- Show error message on error
- Avoid duplicate code by using same component to show proposal discussion and topic item replies

**Comment by ChaituVR** [comment]
> Make ui same as proposal discussion (centered content)

It is mentioned In spec to align to the left https://github.com/snapshot-labs/workflow/issues/192#issue-2499630165

**Comment by ChaituVR** [comment]
> Show error message on error

The error message was not shown on the proposal discussion, and space discussions topic list, should we also add there?

**Comment by wa0x6e** [comment]
> > Show error message on error
> 
> The error message was not shown on the proposal discussion, and space discussions topic list, should we also add there?

Ouch, missed the align left part

**Comment by wa0x6e** [comment]
> > Show error message on error
> 
> The error message was not shown on the proposal discussion, and space discussions topic list, should we also add there?

We could in another PR, since unrelated to this one

**Comment by bonustrack** [comment]
The title line height should be the same as the proposal title

**Comment by ChaituVR** [comment]
Changed it to get title and posts in single call (using loadSingleTopic)

**Pull request merged**